app/src/behaviors/behavior_hold_tap.c

```diff
@@ -483,7 +496,11 @@ static void decide_retro_tap(struct active_hold_tap *hold_tap) {
     if (hold_tap->status == STATUS_HOLD_TIMER) {
         release_binding(hold_tap);
         LOG_DBG("%d retro tap", hold_tap->position);
-        hold_tap->status = STATUS_TAP;
+        if (strcmp(hold_tap->config->retro_tap_behavior, "") == 0) {
```

Comment:
I would make the 'retro_tap_behavior' fall back to the tap behavior. Then you don't need this conditional.